MobileDevProject/ViewController.swift

```diff
@@ -123,10 +124,22 @@ class ViewController: UIViewController, CLLocationManagerDelegate {
                 }
 
                 if isNavigating {
-                    if beacons[0].minor == navigatingBeacon.value && beacons[0].proximity == CLProximity.immediate {
+                    if beacons[0].minor == navigatingBeacon.value && beacons[0].proximity == CLProximity.near {
                         // we have arrived
                         isNavigating = false
-                        print("123223")
+                        // vibrate
+                        AudioServicesPlayAlertSound(SystemSoundID(kSystemSoundID_Vibrate))
```

Comment:
What are you pointing out? 
I chose to add the vibration because I assume its not intended for the user to be staring at the phone, if theres just an alert box then they might miss their destination